pavlova/parsers.py

```diff
@@ -162,6 +162,9 @@ def parse_input(self,
             raise TypeError('Unions of this type are not allowed')
 
         real_field_type = field_type.__args__[0]
+        # fast exit when field_type is Optional and input_value is None
+        if input_value is None:
+            return field_type.__args__[1]()
```

Comment:
Should this be `return None`?

Reply:
I guess I could change it to
```
from typing import cast
return cast(T, None)
```
Would this be a better solution?

Reply:
No, the original way is fine